src/Indicator.vala

```diff
@@ -191,7 +192,7 @@ public class Notifications.Indicator : Wingpanel.Indicator {
 
     private void set_display_icon_name () {
         var dynamic_icon_style_context = dynamic_icon.get_style_context ();
-        if (NotifySettings.get_instance ().do_not_disturb) {
+        if (Indicator.notify_settings.get_boolean ("do-not-disturb")) {
```

Comment:
```suggestion
        if (notify_settings.get_boolean ("do-not-disturb")) {
```

Is there any reason you explicit the namespace `Indicator` although it's in the same class?

Reply:
Just copy/paste error. Thanks for noticing :)